.travis.yml

```
@@ -127,7 +131,9 @@ before_install:
      #  Force ARGS to correct prefix, etc for docker image
      export ARGS="--prefix=/usr --sysconfdir=/etc --with-systemdsystemunitdir=/etc/systemd/system --localstatedir=/var --with-flux-security --enable-caliper"
   fi
-
+ - docker pull minio/minio
+ - docker run -d -p 9000:9000 minio/minio server /data
```

```suggestion
 - if test -n "$S3_HOSTNAME"; then docker run -d -p 9000:9000 minio/minio server /data; fi
```

This will start the minio docker container only if S3_HOSTNAME is set in environment (untested though!). That way we are not running minio when we do not need to be.